alembic/versions/34b8f4307555_create_roles_to_api_table.py

```diff
@@ -0,0 +1,62 @@
+"""create roles_to_api table
+
+Revision ID: 34b8f4307555
+Revises: 0b877ab8221c
+Create Date: 2021-09-22 13:51:15.237803
+
+"""
+
+# revision identifiers, used by Alembic.
+import datetime
+
+from sqlalchemy import orm, text
+
+revision = '34b8f4307555'
+down_revision = '0b877ab8221c'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_table(
+        "roles_to_api",
```

Comment:
I'm not sure what the name `roles_to_api` means

Reply:
I think roles, and the permissions of each role, and mapping of users to roles should be set as configuration (i.e. can be changed without logic change), as they change frequently.
Setting tables / fields with code is common when working with alembic.

Reply:
@atalyaalon What is your opinion?